lib/glific/processor/consumer_flow.ex

```diff
@@ -42,33 +42,39 @@ defmodule Glific.Processor.ConsumerFlow do
       do: FlowContext.mark_flows_complete(message.contact_id)
 
     context = FlowContext.active_context(message.contact_id)
-    # if we are in a flow and the flow is set to ignore keywords
+
```

Comment:
we should look at restructuring this function and simplify it. seems like too many conditionals and cases